### ISSUE TITLE
Auto start the daemon

### ISSUE THREAD

**Issue by Mihailoff** [skymill/automated-ebs-snapshots#16]
What is the best practice to ensure that daemon is running all the time?

**Comment by Mihailoff** [comment]
Based on init.d skeleton https://github.com/jasonblewis/sample-service-script I've created a service for automated-ebs-snapshots https://gist.github.com/Mihailoff/c4eb7f9435f48499b66b

`stop/start/status` commands work fine, the only thing is that after system reboot pid file remains in system and doesn't permit to start the service. Any hints?

**Comment by sebdah** [comment]
Maybe you could implement a start force option to remove the pid file. Our
write another script that removes it on server boot time?

**Comment by mikedunn1** [comment]
I had issues keeping the daemon running when either rebooting the machine or losing the internet connection. Never resolved it, so I began executing --run via crontab. Works great. Only hiccup is, when I scheduled the cron job to run every hour, it wouldn't always create a snapshot because the time elapsed since the previous snapshot was usually slightly less than 3600 seconds (due to the amount of time it takes AWS to actually create the snapshot). I run it every two hours.
